pkg/admission/customresourcevalidation/customresourcevalidationregistration/cr_validation_registration.go

```diff
@@ -25,6 +26,7 @@ var AllCustomResourceValidators = []string{
 	config.PluginName,
 	scheduler.PluginName,
 	clusterresourcequota.PluginName,
+	securitycontextconstraints.PluginName,
```

Comment:
this line.  move to another pull.

Reply:
I'm assuming you mean the admission plugin changes not just the single line.